content/en/docs/Advanced/sys-extensions.md

```diff
@@ -0,0 +1,156 @@
+---
+title: "Extending the system with systemd extensions"
+linkTitle: "Extending the system with systemd extensions"
+weight: 3
+---
+
+
+{{% alert title="Warning" %}}
+This feature is in preview state and only available in Kairos v3.1.x releases and alphas.
+Please check the section "Known issues" at the bottom for more information.
+{{% /alert %}}
+
+
+{{% alert title="Warning" %}}
+This feature is only available for Trusted Boot images. See the [Trusted Boot documentation]({{%relref "/docs/Architecture/trustedboot" %}}) for more information.
```

Comment:
are they Trusted Boot images? don't we call them UKI images? again more in doubt than saying it should change

Reply:
see answer avobe